extensions/amp-analytics/0.1/events.js

```diff
@@ -1279,19 +1293,32 @@ export class VideoEventTracker extends EventTracker {
   /** @override */
   add(context, eventType, config, listener) {
     const videoSpec = config['videoSpec'] || {};
-    const selector = config['selector'] || videoSpec['selector'];
-    const selectionMethod = config['selectionMethod'] || null;
-    const targetReady = this.root.getElement(
-      context,
-      selector,
-      selectionMethod
+    const selectorValue = userAssert(
+      config['selector'] || videoSpec['selector'],
+      'Missing required selector on video trigger'
+    );
+    const selectorArray = isArray(selectorValue)
+      ? selectorValue
+      : [selectorValue];
+
+    userAssert(
+      selectorArray.length,
+      'Missing required selector on video trigger'
     );
+    assertUniqueSelectors(selectorArray);
+    const selectionMethod = config['selectionMethod'] || null;
+    const targetReadyPromises = [];
+
+    selectorArray.forEach((item) => {
+      targetReadyPromises.push(
+        this.root.getElements(context, item, selectionMethod)
+      );
+    });
```

Comment:
This is fishy to me. We would previously have a single `target`, so we'd execute `new AnalyticsEvent(target, ...)` where below is now `AnalyticsEvent(target[0], ...)`. We went from a structure of one `target`, to a 2d array of `target[][]`.

This leads me to believe that usage here is incorrect, since `getElements` can take multiple selectors:

https://github.com/ampproject/amphtml/blob/c194d57cc9727a1a59342661533f13f825d7003c/extensions/amp-analytics/0.1/analytics-root.js#L339-L345

So instead, you should use `getElements()` directly and receive an appropriately 1d array.

(I'm renaming to **`targetPromises`** since `ready` tends to have a meaning associated with loading, and **`selectors`** according to my other comment.)

```suggestion
    const targetsPromise = this.root.getElements(
      context,
      selectors,
      selectionMethod
    );
```

Treating the result flattens accordingly, see my corresponding comment.